nl/guides/course-management/README.md

```diff
@@ -0,0 +1,417 @@
+>
+> 
+> `Zichtbaar`
+>
+>
+> > Alle gebruikers zien de [cursus](#cursus) in het
+> > [cursusoverzicht](#cursusoverzicht).
+> > Ze kunnen ook naar de
+> > [cursuspagina](/nl/for-students#cursuspagina)
+> > navigeren en zich daar eventueel voor de cursus
+> > [registreren](/nl/for-students#cursus-registreren).
+>
+> `Verborgen`
+>
+> > Alleen [cursusbeheerders](#cursusbeheerder) zien de [cursus](#cursus) in het [cursusoverzicht](#cursusoverzicht) en op hun [startpagina](/nl/for-students#startpagina). Er staat een icoontje bij om hen er op te wijzen dat andere gebruikers de cursus daar niet kunnen zien. Dit icoontje is ook te vinden op de [cursuspagina](/nl/for-students#cursuspagina) zelf. Enkel geregistreerde gebruikers kunnen naar deze pagina navigeren. Andere gebruikers kunnen zich enkel voor de cursus registreren als ze de [registratielink](#registratielink) gebruiken.
+> >
+> > ![image](./staff.courses_hidden_course.png)
+
+`Registratieprocedure`
+
+> De registratieprocedure bepaalt of en hoe gebruikers zich voor de
+> [cursus](#cursus) kunnen [registreren](/nl/for-students#cursus-registreren). Voor
+> deze eigenschap kunnen de volgende waarden ingesteld worden:
+>
+> `Open`
+>
+> > Gebruikers kunnen zich voor de [cursus](#cursus) [registreren](/nl/for-students#cursus-registreren) zonder expliciete goedkeuring van een [cursusbeheerder](#cursusbeheerder).
+>
+> `Gemodereerd`
+>
+> > Gebruikers kunnen een
+> > [registratieverzoek](/nl/for-students#registratieverzoek)
+> > [indienen](/nl/for-students#registratieverzoek-indienen) voor de [cursus](#cursus)
+> > maar zijn pas [geregistreerd](/nl/for-students#cursus-registreren) als een
+> > [cursusbeheerder](#cursusbeheerder)
+> > hun registratieverzoek heeft
+> > [goedgekeurd](#registratieverzoek-goedkeuren).
+>
+> `Gesloten`
+>
+> > Gebruikers kunnen zich niet meer voor de
+> > [cursus](#cursus) [registreren](/nl/for-students#cursus-registreren).
+>
+> ::: tip Belangrijk
+>
+> Als je de registratieprocedure aanpast dan blijven bestaande
+> [registraties](/nl/for-students#cursus-registreren) voor
+> de [cursus](#cursus) gelden en blijven
+> [registratieverzoeken](/nl/for-students#registratieverzoek) voor de cursus openstaan. Je moet zelf de bestaande
+> [registratiestatus](#registratiestatus) van [cursusgebruikers](#cursusgebruikers)
+> [aanpassen](#registratiestatus-aanpassen).
+> :::
+
+`Beschrijving`
+
+> Een optionele beschrijving die bovenaan de
+> [cursuspagina](#<cursuspagina>) wordt
+> weergegeven. Voor het opmaken van de beschrijving kan je gebruikmaken van Markdown.
+
+>
+> ::: tip
+> <h1 id="beschrijving-markdown"></h1>
+
+> De beschrijving van eigenschappen die aangeduid worden met het
+> Markdown-logo kan geschreven worden met behulp van
+> [Markdown](https://en.wikipedia.org/wiki/Markdown). Dodona maakt voor
+> de weergave van Markdown gebruik van
+> [kramdown](https://kramdown.gettalong.org) waardoor heel wat
+> uitbreidingen van de standaard Markdown ondersteund worden.
+> :::
```

Comment:
This is broken in the page display. Remove the tip, integrate it in the previous paragraph and link the our documentation about formatting descriptions: references/exercise-description/